src/less/notice/base/notice.less

```diff
@@ -49,3 +49,13 @@ button.page-notice__close {
         .background-icon-base;
     }
 }
+
+a.page-notice,
```

Comment:
I moved these rules into the "base" in order to fix the CTA Page Notice example in DS6 docs.